admin/metabox/class-metabox-section-react.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin
+ */
+
+/**
+ * Generates and displays the React root element for a metabox section.
+ */
+class WPSEO_Metabox_Section_React implements WPSEO_Metabox_Section {
+
+	/**
+	 * Name of the section.
+	 *
+	 * @var string
+	 */
+	public $name;
+
+	/**
+	 * Content to use above the React root node.
+	 *
+	 * @var string
+	 */
+	public $content;
+
+	/**
+	 * Content to use for the link.
+	 *
+	 * @var string
+	 */
+	private $link_content;
+
+	/**
+	 * Link title to use.
+	 *
+	 * @var string
+	 */
+	private $link_title;
```

Comment:
I think we're not using a title attribute any longer, and we've kept it in the previous implementation even if it's not used. We're not going to use title attributes so I'd consider to remove this.